src/state/slices/common-selectors.ts

```diff
@@ -29,12 +30,11 @@ import {
   selectSearchQueryFromFilter,
 } from '@/state/selectors'
 
-export const selectWalletId = (state: ReduxState) => state.portfolio.connectedWallet?.id
-export const selectWalletName = (state: ReduxState) => state.portfolio.connectedWallet?.name
-export const selectIsWalletConnected = (state: ReduxState) =>
-  state.portfolio.connectedWallet !== undefined
-export const selectWalletSupportedChainIds = (state: ReduxState) =>
-  state.portfolio.connectedWallet?.supportedChainIds ?? []
+export const selectWalletId = portfolio.selectors.selectWalletId
+export const selectWalletName = portfolio.selectors.selectWalletName
+export const selectIsWalletConnected = portfolio.selectors.selectIsWalletConnected
+export const selectWalletSupportedChainIds = portfolio.selectors.selectWalletSupportedChainIds
```

Comment:
For this and similar diffs: kept in the same loc to keep diff tidier - ideally, we'd export that directly from the slice but consuming the new location in all would make for a much, much bigger diff.